charts/vineyard/templates/worker/statefulset.yaml

```diff
@@ -161,6 +176,8 @@ spec:
         hostPath:
           path: /runtime-mnt/vineyard/{{ .Release.Namespace }}/{{ $fullName }}
```

Comment:
I think `/runtime-mnt` is configurable.  Try to avoid hard code this directory.

Reply:
Make sense to me.